src/client/styles/components/EditionsList.scss

```diff
@@ -1,6 +1,6 @@
 .nypl-editions-table {
   width: 100%;
-  margin: 0 0 20px 0;
+  margin: 0;
```

Comment:
The table is now running into the NYPL footer in some (all?) instances with the `mainContent` padding removed as well. Not sure where that padding should live, but it seems like there should be something there

Reply:
That's true, I didn't see it. I just made a commit to solve that